sdk/keyvault/keyvault/ci.yml

```diff
@@ -1,17 +0,0 @@
-# DO NOT EDIT THIS FILE
```

Comment:
Was it intended to remove this file in this PR?

Reply:
Yes, it was discussed offline to remove this as it is no longer needed and the pipeline will always fail. I can separate this out into another PR first and follow up with this one if that's preferred.